examples/build.gradle

```diff
@@ -30,7 +30,7 @@ android {
     defaultConfig {
         applicationId "com.mapbox.navigation.examples"
         minSdkVersion androidVersions.minSdkVersion
-        targetSdkVersion androidVersions.targetSdkVersion
+        targetSdkVersion 31
```

Comment:
Why don't we introduce a new item in `dependencies.gradle` that `examples` and `qa` module could share?

Reply:
We can do that, but I was going to update `targetSdkVersion` in other modules soon, so I decided not to introduce a temporary property.